src/messaging/ExchangeContext.cpp

```diff
@@ -628,6 +631,9 @@ CHIP_ERROR ExchangeContext::HandleMessage(uint32_t messageCounter, const Payload
         // If the context was expecting a response to a previously sent message, this message
         // is implicitly that response.
         SetResponseExpected(false);
+
+        // If we received the expected response, we don't if the peer is still active after having sent the response
```

Comment:
"we don't if" ? Seems like the sentence needs to be reworked to say what it meant to say.